### PR TITLE
Fix trivial new line #trivial

### PR DESCRIPTION
**Pull request by JosephSalomon**
fixed '\n' to actually write on a new line and not show \n on the screen. Also checks to see if the PR was declared trivial or if it was tested issue #103

### CODE REVIEW

**Comment by Huddie** [comment]
@ymoskovits hahah I agree that 2nd message of  "Thanks for remembering to add tests!" doesn't always make sense :D. @JosephSalomon I agree with @ymoskovits to break it into 3 cases.

File: Dangerfile [hunk 1]
```diff
@@ -43,10 +43,15 @@ end
...
-	warn('It appears that you forgot to add a Unit Test to the test file.\n Please add a test and upload the new version.\n The test file can currently be found at: ./src/tests/tests.py\n If this is a "trivial" change add #trivial to the title')
-else
+
+if forgot_tests and not declared_trivial
+	warn("It appears that you forgot to add a Unit Test to the test file.\r\n Please add a test and upload the new version.\r\n The test file can currently be found at: ./src/tests/tests.py\r\n If this is a 'trivial' change add #trivial to the title")
+
+elif not forgot_tests
 	message("Thanks for remembering to add a test!")
+
+elif declared_trivial
```
Comment (Huddie) [inline Dangerfile]: You can use a ternary and a 1 else

```ruby
if forgot_..... and not ... 
      .....
else
      msg = not forgot_tests ? "Thanks for remembering to add a test!" : "Thanks for remembering to declare trivial!"
     message(msg)
end
```

**Comment by Huddie** [comment]
Anyone see the issue some parenthesis problem.

File: Dangerfile [hunk 2]
```diff
@@ -43,10 +43,13 @@ end
...
-	warn('It appears that you forgot to add a Unit Test to the test file.\n Please add a test and upload the new version.\n The test file can currently be found at: ./src/tests/tests.py\n If this is a "trivial" change add #trivial to the title')
+
+if forgot_tests and not declared_trivial
+	warn("It appears that you forgot to add a Unit Test to the test file.\r\n Please add a test and upload the new version.\r\n The test file can currently be found at: ./src/tests/tests.py\r\n If this is a 'trivial' change add #trivial to the title")
+
 else
-	message("Thanks for remembering to add a test!")
+      msg = not forgot_tests ? "Thanks for remembering to add a test!" : "Thanks for remembering to declare trivial!"
+     message(msg)
 end
```
Comment (ymoskovits) [inline Dangerfile]: Try using ! instead of not here. I think that should fix it if it was causing a problem before.

**Comment by Huddie** [comment]
@ymoskovits Nice catch. @JosephSalomon I'll review, should be good (Check out CUNY-Bot! he's working well!!!) sweet fix

**Comment by ymoskovits** [comment]
> @ymoskovits Nice catch. @JosephSalomon I'll review, should be good (Check out CUNY-Bot! he's working well!!!) sweet fix

That was a cuny-bot catch?

**Comment by Huddie** [comment]
No, I was just saying CUNY-Bot says "Thanks for remembering to declare trivial!" so hes back up and running properly again.

**Comment by Huddie** [comment]
@ymoskovits just approve the PR if it looks good to you

**Pull request merged**